assets/models/system/mmd-3x-vfnet_x101-64x4d-mdconv-c3-c5_fpn_ms-2x_coco/spec.yaml

```diff
@@ -21,12 +26,96 @@ properties:
     Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4,
     Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4,
     Standard_ND40rs_v2
-  model_id: mmd-3x-vfnet_x101-64x4d-mdconv-c3-c5_fpn_ms-2x_coco
 tags:
   Preview: ''
+  SharedComputeCapacityEnabled: ''
+  open_mmlab_model_id: mmd-3x-vfnet_x101-64x4d-mdconv-c3-c5_fpn_ms-2x_coco
+  training_dataset: COCO
```

Comment:
At some places we have "coco" and at other places we have "COCO". I think, better to align them.

Reply:
done